utils/global-state-update-gen/src/generic/testing.rs

```diff
@@ -781,8 +781,8 @@ fn should_replace_one_validator() {
     assert!(!bid_write.inactive());
 
     // check that the bid purse for the new validator has been created with the correct amount
-    update.assert_written_purse_is_unit(bid_write.bonding_purse());
-    update.assert_written_balance(bid_write.bonding_purse(), 102);
+    update.assert_written_purse_is_unit(bid_write.bonding_purse().unwrap());
+    update.assert_written_balance(bid_write.bonding_purse().unwrap(), 102);
```

Comment:
as a side note, in tests we prefer to use .expect("message of what was expected") instead of .unwrap()